.github/actions/ubuntu_dependencies/action.yml

```diff
@@ -0,0 +1,36 @@
+name: fastddsspy_dependencies_ubuntu
+description: Common first step for all Fast DDS Spy jobs. It checkout repository, downloads dependencies and install required packages.
+
+inputs:
+
+  ubuntu_version:
+    description: Specify runs-on machine to download specific artifact
+    required: true
+
+  dependencies_artifact_postfix:
+    description: Specify artifact postfix in case it wants to use a manual one
+    required: false
+    default: _nightly
+
+  target_workspace:
+    description: Specify directory to download dependencies
+    required: false
+    default: './install'
+
+runs:
+  using: composite
+  steps:
+
+    - name: Install Fast DDS dependencies
+      uses: jparisu/eProsima-CI/ubuntu/install_fast_dependencies@main
+
+    - name: Install yaml cpp dependency
```

Comment:
Already done when installing DDS Pipe?

Reply:
Noup. The ddspipe comes with every **eProsima** dependency installed. But the system is reset and is not reused from artifact, neither it can be done from a previous step or image :disappointed: